test/easyconfigs/easyconfigs.py

```diff
@@ -575,7 +575,7 @@ def get_deps_for(ec):
 
         # restrict to checking dependencies of easyconfigs using common toolchains (start with 2018a)
         # and GCCcore subtoolchain for common toolchains, starting with GCCcore 7.x
-        for pattern in ['201[89][ab]', '20[2-9][0-9][ab]', r'GCCcore-([7-9]|1[0-9])\.[0-9]']:
+        for pattern in ['201[89][ab]', '20[2-9][0-9][ab]', r'GCCcore-([7-9]|[1-9][0-9])\.[0-9]']:
```

Comment:
Maybe do the same for the 20xx? This saves a full iteration over ALL ECs
```suggestion
        for pattern in ['20(1[89]|[2-9][0-9])[ab]', r'GCCcore-([7-9]|[1-9][0-9])\.[0-9]']:
```

Reply:
I like the idea!
I have opened an PR to improve this: https://github.com/easybuilders/easybuild-easyconfigs/pull/13251